tests/new_integration_tests/test_repos.py

```diff
@@ -356,6 +357,29 @@ def test_clone_all_repos(self, platform_url, with_student_repos, tmp_path):
             STUDENT_TEAMS, TEMPLATE_REPO_NAMES, tmp_path
         )
 
+    def test_clone_all_repos_flat(
+        self, platform_url, with_student_repos, tmp_path
+    ):
+        """Test that cloning with flat directory layout results in all
+        repositories ending up in the current working directory.
+        """
+        expected_dirnames = plug.generate_repo_names(
+            STUDENT_TEAMS, TEMPLATE_REPO_NAMES
+        )
+
+        funcs.run_repobee(
+            f"repos clone -a {TEMPLATE_REPOS_ARG} "
+            f"--base-url {platform_url} "
+            "--directory-layout "
+            f"{_repobee.command.repos.DirectoryLayout.FLAT.value}",
```

Comment:
Here's where you call `.value` instead of the string function you created

Reply:
I probably made this test before implementing the string function. The `.value` is redundant here.